stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -104,8 +105,9 @@ spec:
           failureThreshold: {{ .Values.master.livenessProbe.failureThreshold }}
           exec:
             command:
-            - redis-cli
-            - ping
+            - sh
+            - -c
+            - /health/ping_local.sh
```

Comment:
Upon further testing, I'm seeing an issue with this causing the master pod to restart. Slaves do not seem to be affected... please hold off on approval for now.

Reply:
Okay, fixed. `REDIS_PORT` was not set in the master deployment due to a misplaced `else`.